lib/pbench/server/api/resources/users_api.py

```diff
@@ -7,25 +7,33 @@
 from email_validator import EmailNotValidError
 from sqlalchemy.exc import SQLAlchemyError, IntegrityError
 from typing import NamedTuple
+from logging import Logger
+from pbench.server import PbenchServerConfig
 
 from pbench.server.api.resources import (
```

Comment:
Line 11 belongs with the "pbench" group of imports.

Ideally, line 10 would be inserted into its group in sorted order...but that would require sorting the group....  (And, the "pbench group" could use sorting, as well....)